source/databricks/tests/integration/calculator/balance_fixing_total_production/test_get_metering_points_df.py

```diff
@@ -450,3 +450,31 @@ def test__metering_points_are_periodized_by_effective_date(
 
     # Assert
     assert (actual_df.count() == 1) == expected
+
+
+def test__duplicate_events_does_not_affect_amount_of_periods(
```

Comment:
I don't think you should only test that it doesn't affect _the amount of_ periods. The periods should still be equal. So I suggest the more precise name:
```suggestion
def test__duplicate_events_does_not_affect_periods(
```